unstructured/ingest/interfaces.py

```diff
@@ -13,17 +13,19 @@
 
 import requests
 from dataclasses_json import DataClassJsonMixin
+from dataclasses_json.core import Json, _asdict, _decode_dataclass
 
 from unstructured.chunking.title import chunk_by_title
 from unstructured.documents.elements import DataSourceMetadata
 from unstructured.embed.interfaces import BaseEmbeddingEncoder, Element
 from unstructured.embed.openai import OpenAIEmbeddingEncoder
 from unstructured.ingest.error import PartitionError, SourceConnectionError
-from unstructured.ingest.ingest_doc_json_mixin import IngestDocJsonMixin
```

Comment:
...do we still need unstructured.ingest.ingest_doc_json_mixin in this case? or okay to remove it?

Reply:
This is waiting for the other PR to be merged in. I just copied it over for now.